imports/plugins/core/ui/client/components/app/coreReactionApp.js

```diff
@@ -0,0 +1,106 @@
+import { Switch } from "react-router-dom";
```

Comment:
Simply call is app.js and class name App. core and reaction are not necessary. Everywhere its called CoreReactionApp should just be App.